docs/DeveloperGuide.adoc

```diff
@@ -23,7 +23,7 @@ By: `Team Heart²`      Since: `Aug 2018`      Licence: `MIT`
 
 Welcome to *_Heart²_*! _Heart²_ is a desktop software intended to make the job of wedding planning agencies simpler.
 It provides simple yet powerful features to efficiently manage clients' and agency companies' profiles.
-Users can find suitable wedding service providers for couples using just a few keystrokes with our enterprise featureset.
+Users can find suitable vendors providing services for couples using just a few keystrokes with our enterprise featureset.
```

Comment:
I think we can go with "vendors providing wedding services" haha slightly clearer